rero_ils/modules/items/listener.py

```diff
@@ -57,3 +59,7 @@ def enrich_item_data(sender, json=None, record=None, index=None,
         # Document type
         document = Document.get_record_by_pid(json['document']['pid'])
         json['document']['document_type'] = document['type']
+
+        # Current pending requests
+        json['current_pending_requests'] = \
+            LoansSearch().get_pending_request_count(item.pid)
```

Comment:
`json['current_pending_requests '] = Item.get_requests(output='count')`

and you could remove your `LoansSearch().get_pending_request_count()` method.